pyocd/target/builtin/target_HC32F160.py

```diff
@@ -0,0 +1,179 @@
+# pyOCD debugger
+# Copyright (c) 2021 Huada Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ...flash.flash import Flash
```

Comment:
Could you please remove this unused import caught by LGTM? Sorry I didn't notice it before!